helpers/channels/triggerTypingIndicator.ts

```diff
@@ -5,6 +5,6 @@ import type { Bot } from "../../bot.ts";
  * However, if a bot is responding to a command and expects the computation to take a few seconds,
  * this endpoint may be called to let the user know that the bot is processing their message.
  */
-export async function startTyping(bot: Bot, channelId: bigint): Promise<void> {
+export async function triggerTypingIndicator(bot: Bot, channelId: bigint): Promise<void> {
```

Comment:
Why not keep an alias of `startTyping` for this?

Reply:
export const startTyping = triggerTypingIndicator;

Reply:
Added alias in latest commit.